common/lifecyleMgr.go

```diff
@@ -344,6 +345,18 @@ func (lcm *lifecycleMgr) Dryrun(o OutputBuilder) {
 	}
 }
 
+func (lcm *lifecycleMgr) Output(o OutputBuilder) {
+	om := ""
+	if o != nil {
+		om = o(lcm.outputFormat)
+	}
+
+	lcm.msgQueue <- outputMessage{
+		msgContent: om,
+		msgType:    eOutputMessageType.Info(),
```

Comment:
instead of the current json implementation, could I just set msgType to ListObject vs ListSummary? Or does this expect only Info vs Debug etc

Reply:
I believe new enum entries could be used, yeah-- I think processTextOutput will probably have words to say about that unless added to the switch, FYI.